vsphere/vsphere.go

```diff
@@ -149,35 +152,95 @@ func (ops *vsphereOps) Create(opts interface{}, labels map[string]string) (inter
 
 	volumeOptions.Datastore = datastore
 
-	diskBasePath := filepath.Clean(ds.Path(diskDirectory)) + "/"
-	err = ds.CreateDirectory(ctx, diskBasePath, false)
-	if err != nil && err != vclib.ErrFileAlreadyExist {
-		logrus.Errorf("Cannot create dir %#v. err %s", diskBasePath, err)
-		return nil, err
-	}
+	var (
+		trueVar = true
```

Comment:
nit: this could be renamed to keepDiskAfterDeleteVm

Reply:
done